File: elephant_mocha/Extra/Elephant.js
class Elephant {
  constructor (theElephantOwner, newHeight, newColor, newSpecies, newGender) {
    this.myElephantOwner = theElephantOwner
    this.height = newHeight
    this.color = newColor
    this.species = newSpecies
    this.gender = newGender
  }
  toString () {
    return '\ta ' + this.color + ' ' + this.species 
           + ' (' + this.gender + ') ' + 'with a height of ' + this.height + 'ft'
  }
}

File: elephant_python/elephant.py
class Elephant:
    def __init__(self, new_height, new_color, new_species, new_gender, the_owner):
        self.my_owner = the_owner
        self.height = new_height
        self.color = new_color
        self.species = new_species
        self.gender = new_gender

    def __str__(self):
        return f'{self.color} {self.species} [{self.gender}] with the height of {self.height}'
